pkg/ingress/apisix_cluster_config.go

```diff
@@ -277,3 +277,21 @@ func (c *apisixClusterConfigController) onDelete(obj interface{}) {
 
 	c.controller.MetricsCollector.IncrEvents("clusterConfig", "delete")
 }
+
+func (c *apisixClusterConfigController) ResourceSync() {
+	objs := c.controller.apisixClusterConfigInformer.GetIndexer().List()
+	for _, obj := range objs {
+		key, err := cache.MetaNamespaceKeyFunc(obj)
+		if err != nil {
+			log.Errorf("ApisixClusterConfig sync failed, found ApisixClusterConfig resource with bad meta namespace key: %s", err)
+			continue
+		}
+		if !c.controller.isWatchingNamespace(key) {
+			continue
+		}
+		c.workqueue.Add(&types.Event{
+			Type:   types.EventAdd,
```

Comment:
Why did you choose to use the `types.EventAdd` event type?

Reply:
* If ` types Eventupdate `, the controller needs to compare with the old objects during processing, which does not take effect in `apisixroute` `upstream`, and the logic needs to be modified.
* For example, the object of apifix is deleted manually, or the resource conversion fails. The semantics is equivalent to retry. I think it is appropriate to use `types.eventadd`.